.grit/patterns/universal/sort_fn.md

```diff
@@ -0,0 +1,23 @@
+# Sort function
+
+The `sort($list)` function sorts a list of items in ascending order, based on the lexical order of the items.
```

Comment:
Could be good to include test case for integers to make sure there's no string casting that produces a lexical sort

`[2, 10, 3]`

Reply:
This is actually intentionally a lexical sort.